spec/rspec/matchers/built_in/yield_spec.rb

```diff
@@ -617,6 +674,16 @@ def each_arg(*args, &block)
       }.not_to yield_successive_args(a_string_matching(/foo/), a_string_matching(/bar/))
     end
 
+    it 'passes when the successively yielded args match the matchers at yield time only' do
```

Comment:
I chose these names to match the ones above, but they don't actually look right to me. Shouldn't they be the other way round — e.g.

```ruby
it 'passes when the successively yielded args don't match the matchers'
```